### PR TITLE
change install page and add link to integration page in the platform

### PR DESCRIPTION
**Pull request by guymenahem**

### CODE REVIEW

File: docs/Learn/Komodor-Agent.md [hunk 1]
```diff
@@ -2,20 +2,13 @@
 
 ## Installation
 
-!!! Note ""
-    Before installing the agent, get your API key from the Integration page
 
-![API Key Location](./img/api_key_location.png)
+### Get an API Key
+The API key of each account is available in the platform [Integration page](https://app.komodor.com/main/integration).
```
Comment (olegsu) [inline docs/Learn/Komodor-Agent.md:7]: I suggest to rephrase to something like
`Tha API key can be found in the Integration page`

File: docs/Learn/Komodor-Agent.md [hunk 2]
```diff
@@ -2,20 +2,13 @@
...
+![API Key Location](./img/api_key_location.png)
 
-```bash
-export KOMOKW_API_KEY= # API KEY Required
-export KOMOKW_CLUSTER_NAME= # Optional
-kubectl create ns komodor
-kubectl apply -n komodor -k https://github.com/komodorio/helm-charts/manifests/overlays/full/?ref=master
-```
-### Helm
+###Install Using Helm
```
Comment (olegsu) [inline docs/Learn/Komodor-Agent.md:11]: Remove `install using` and add space after the `###`

File: docs/Learn/Komodor-Agent.md [hunk 3]
```diff
@@ -28,6 +21,16 @@ helm upgrade --install k8s-watcher komodorio/k8s-watcher \
 
 ```
 
+### Install Using Kustomize
```
Comment (olegsu) [inline docs/Learn/Komodor-Agent.md:24]: Remove install using

**Review by olegsu — CHANGES_REQUESTED** [review]

File: docs/Learn/Komodor-Agent.md (hunk 1 above)
Comment (guymenahem) [reply docs/Learn/Komodor-Agent.md]: changed

File: docs/Learn/Komodor-Agent.md (hunk 3 above)
Comment (guymenahem) [reply docs/Learn/Komodor-Agent.md]: changed

File: docs/Learn/Komodor-Agent.md (hunk 2 above)
Comment (guymenahem) [reply docs/Learn/Komodor-Agent.md]: changed

**Comment by olegsu** [comment]
@guymenahem please rebase master to have the latest changes

**Review by olegsu — APPROVED** [review]

**Pull request merged**